BasementRenovator.py

```diff
@@ -6017,9 +6022,10 @@ def writeTestData(self, folder, testType, floorInfo, testRooms):
     StageType = {floorInfo.get('StageType')},
     StageName = {strFix(floorInfo.get('Name'))},
     IsModStage = {floorInfo.get('BaseGamePath') is None and 'true' or 'false'},
+    DisableUI = {settings.value("DisableInGameUI")},
```

Comment:
can you make this write `true` or `false` explicitly? Idk what this will do when the value isn't set at all.

Reply:
Implemented this and the keyboard toggle